mmcv/ops/csrc/common/cuda/voxelization_cuda_kernel.cuh

```diff
@@ -122,7 +122,7 @@ __global__ void point_to_voxelidx_kernel(const T_int* coor,
           point_to_pointidx[index] = i;
         } else if (num >= max_points) {
           // out of boundary
-          return;
+          break;
```

Comment:
break is not enough, you should mark it with a flag and continue in the outer loop.

Reply:
Right, but if num >= maxpoints and break loop, and then check `if (num == 0)` and `if (num < max_points)` is invalid, so I think break is enough.

Reply:
Sure, I see.